compiler/modules/bir/bir.bal

```diff
@@ -663,6 +691,14 @@ public type CallIndirectInsn readonly & record {|
     boolean restParamIsList;
 |};
 
+# Create a closure by enclosing the first operand with the remaining operands.
```

Comment:
Create a function value from an AnonFunction.
The operands are the values for the captured registers.

Reply:
Fixed